docs/source/using-federation/apollo-gateway-setup.mdx

```diff
@@ -0,0 +1,458 @@
+---
+title: Implementing a gateway with Apollo Server
+description: Using a Node.js gateway as your graph router
+---
+
+import FederationArchitecture from '../shared/diagrams/federation-architecture.mdx';
+
+After you set up at least one federation-ready [subgraph](/federation/building-supergraphs/subgraphs-overview), you can configure a **graph router** (also known as a gateway) to sit in front of your subgraphs.
```

Comment:
This new article is moving over the content from the  "[Node.js gateway setup](https://www.apollographql.com/docs/federation/building-supergraphs/router#nodejs-gateway-setup)" section downwards in the [Router/gateway article](https://www.apollographql.com/docs/federation/building-supergraphs/router#nodejs-gateway-setup). 

I added this small intro section, updated links, and added `MultiCodeBlocks`.